### ISSUE TITLE
bug: Dataflow does not show flow_input.iter

### ISSUE THREAD

**Issue by wasnertobias** [windmill-labs/windmill#3989]
### Describe the bug

Dataflow does not show flow_input.iter

### To reproduce

-

### Expected behavior

-

**Comment by rubenfiszel** [comment]
Could you share more details please

**Comment by wasnertobias** [comment]
The dataflow feature in the flow editor does not show any lines for the usage of flow_input.iter - so when using a loop to iterate over an array the usage of specific key on the single items is hard to follow.

[{"a": 1, "b": "second attribute"}, {"a":543, "b": "some value"}] => Result of step a

Step a is then iterated over in a flow loop. Now check with the dataflow feature where attribute "a" or "b" is used within the loop. => Not possible.

**Comment by rubenfiszel** [comment]
@fatonramadani we're missing passing for while loops
